### ISSUE TITLE
sway floating > spawns at top left corner instead at the middle of the screen

### ISSUE THREAD

**Issue by narutowindy** [swaywm/sway#1229]
hi, i'm using  **sway**  in arch linux,  terminal URxvt is **floating enabled**(no other application is opened) but it **spawns**  at **top+ left** of my screen, *this was not the case in i3*,

**Comment by SirCmpwn** [comment]
https://github.com/SirCmpwn/sway/wiki#how-do-i-report-issues

**Comment by narutowindy** [comment]
sway version 0.13.0

Sway.log > below link
[sway.txt](https://github.com/SirCmpwn/sway/files/1030843/sway.txt)  
[sway config file] (https://github.com/narutowindy/dotfiles/blob/master/sway/config)

**Comment by narutowindy** [comment]
i opened **google-chrome** in one workspace ** without floating** ,  now i tried to open terminal Urxvt ,Now it spawned at the **top+middle** position .

**Comment by narutowindy** [comment]
Something wrong with my issue ?? @SirCmpwn  just take a look at it...

**Comment by SirCmpwn** [comment]
I'll get to it if I have time, but this isn't exactly that important in the grand scheme of things. I encourage you to dig into the code and see if you can find out the issue yourself if you want it done faster.

**Comment by narutowindy** [comment]
i just want to know, this issue only with me or anybody having the same issue? if not i will reinstall sway to get back to it. @SirCmpwn

**Comment by narutowindy** [comment]
i did a fresh reinstall  whole machine with Arch+official sway 0.13 packacge, when i set my URxvt to foating enabled by default,  it still spwans at the **top left corner** .  @SirCmpwn

**Comment by SirCmpwn** [comment]
So look into it.

**Comment by narutowindy** [comment]
do you want me to close the issue??  @SirCmpwn

**Comment by SirCmpwn** [comment]
No, I want you to debug it and send a patch.

**Comment by bearcatsandor** [comment]
If they don't know C, they can't submit a patch. Should they not have reported the bug?

**Comment by SirCmpwn** [comment]
C isn't difficult. Anyone can learn it.

**Comment by kaberc** [comment]
I think you can try to replace this:
```
#:::::::::disable_window_borders::::::::                                                                  
 for_window [class="^.*"] border pixel 0                                                                   
 new_window pixel 1                                                                                        
 new_float  normal   
```
in your config with ```default_border pixel 1``` and report here if it helped

**Comment by narutowindy** [comment]
@kaberc nope its not working ..

**Comment by narutowindy** [comment]
would you consider #1229 for this 0.15 release??

spawning in the floating mode is seems buggy !
"feh, mpv , Urxvt " all seems to be not working as expected in Floating mode.

**Comment by SirCmpwn** [comment]
I would if you sent me a patch.

**Comment by narutowindy** [comment]
i wish i could send you a patch, unfortunately i cant make one.

**Comment by SirCmpwn** [comment]
I bet you could!

**Comment by v0idwalker** [comment]
This conversation was glorious. 
Anyhow, firefox-nightly (60) also spaws its default browser dialogue in the top left corner.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
